### PR TITLE
reimbursement builder now requires people.  It needs tests fixing so …

### PR DESCRIPTION
**Pull request by sbillinge**
@scopatz @CJ-Wright this is ready to merge.  It forces a name to be specified on the command line for reimbursement builder, which is incredibly slow .

### CODE REVIEW

File: regolith/builders/reimbursementbuilder.py [hunk 1]
```diff
@@ -55,103 +62,121 @@ def construct_global_ctx(self):
...
+                            total_amount += item.get("unsegregated_expense", 0)
+                        except TypeError:
+                            if item.get("unsegregated_expense", 0) == 'tbd':
+                                print("WARNING: unsegregated expense in {} is "
+                                      "tbd".format(ex["_id"]))
+                                item["unsegregated_expense"] = 0
+                            else:
+                                sys.exit(
+                                    "ERROR: unsegregated expense in {} is not "
+                                    "a number".format(ex["_id"]))
```
Comment (scopatz) [inline regolith/builders/reimbursementbuilder.py]: You should not use sys.exit in library code.  Please raise an approriate error instead

**Comment by sbillinge** [comment]
thanks @scopatz I fixed that so it reraises the TypeError.  I left the sys.exit in the global-context place.  Is it ok there?

**Comment by scopatz** [comment]
I really don't think we should have any sys.exits

**Comment by scopatz** [comment]
I really don't think we should have any sys.exits

**Comment by sbillinge** [comment]
@scopatz I think this is ready for merge.  

Can we make a release and push it to conda-forge?   This seems to be needed for the  CI to run properly again.

**Comment by scopatz** [comment]
doing a release now

**Pull request merged**